src/script/auth/page/CheckPassword.tsx

```diff
@@ -153,7 +153,6 @@ const CheckPasswordComponent = ({
                   formNoValidate
                   onClick={handleLogin}
                   disabled={isFetching || !password}
-                  showLoading={isFetching}
```

Comment:
This will actually never render with the value `true` since it's already caught in the parent condition (see L147)